luigi/util.py

```diff
@@ -283,21 +283,24 @@ def run(self):
                # ...
     """
 
-    def __init__(self, *tasks_to_inherit):
+    def __init__(self, *tasks_to_inherit, ignore_collisions=[]):
         super(inherits, self).__init__()
         if not tasks_to_inherit:
             raise TypeError("tasks_to_inherit cannot be empty")
 
         self.tasks_to_inherit = tasks_to_inherit
+        self.ignore_collisions = ignore_collisions
```

Comment:
highly recommend
```python
def __init__(self, *tasks_to_inherit, ignore_collisions=None):
    ...
    self.ignore_collisions = ignore_collisions or []
```

Reply:
I agree that was a bad practice. Assuming the mistake was using the mutable default argument, I actually prefer replacing the empty list with a tuple,

```Python
def __init__(self, *tasks_to_inherit, ignore_collisions=()):
    ...
    self.ignore_collisions = ignore_collisions
```
This way we remove the mutable default argument and at the same time allow the argument to communicate to a developer that the expected input is a sequence. The fewer lines the developer has to read before inferring the expected input the better.

Reply:
No complaint from me :)